ui/src/geoviite-design-lib/accordion/accordion.tsx

```diff
@@ -54,17 +57,11 @@ export const Accordion: React.FC<AccordionProps> = ({
                     {subheader && <div className="accordion__subheader">{subheader}</div>}
                 </span>
                 {onVisibilityToggle && (
-                    <span
-                        className={`${styles['accordion__visibility']} ${
-                            visibility ? styles['accordion__visibility--visible'] : ''
-                        }`}
-                        onClick={onVisibilityToggle}>
-                        {fetchingContent ? (
-                            <CircularProgress />
-                        ) : (
-                            <Icons.Eye color={IconColor.INHERIT} />
-                        )}
-                    </span>
+                    <Eye
+                        onVisibilityToggle={onVisibilityToggle}
+                        visibility={visibility}
+                        fetchingContent={fetchingContent}
+                    />
```

Comment:
Mietin, että tämä silmä voisi kannattaa erottaa kokonaan `Accordion`ista ja leipoa vaan `GeometryPlanPanel`in käyttötapauksessa osaksi sen sisältöä. Accordionia käytetään (ainakin muistaakseni) muuallakin, ja niissä käyttötilanteissa silmää ei tarvita